integration_test/test_videoroom/test/support/mustang.ex

```diff
@@ -1,41 +1,45 @@
 defmodule TestMustang do
   use Stampede.Mustang
+  require Logger
 
   @impl true
   def join(browser, options) do
     page = browser |> Playwright.Browser.new_page()
     _response = Playwright.Page.goto(page, options.target_url)
 
-    Process.sleep(500)
-
-    :ok = Playwright.Page.click(page, "[id=#{options.start_button}]")
+    page
+    |> Playwright.Page.locator("[id=#{options.start_button}]")
+    |> Playwright.Locator.click()
```

Comment:
Did you change `Playwright.Page.click` for `Playwright.Page.locator` to use two pipes?

Reply:
The reason was to remove `Proces.sleep(500)`